apps/web/src/utils/cookies.ts

```diff
@@ -43,6 +43,6 @@ export function setNovuOnboardingStepCookie() {
   return novuOnboardedCookie.set('1', {
     expires: ONBOARDING_COOKIE_EXPIRY_DAYS,
     sameSite: 'none',
-    secure: window.location.protocol === 'https',
+    secure: true,
```

Comment:
@ainouzgali `true` doesn't work for localhost environments when server over http. 

Actually the previous check was wrong. An `:` was missing.
`secure: window.location.protocol === 'https:'`